packages/node_modules/@node-red/editor-client/src/js/nodes.js

```diff
@@ -849,40 +849,47 @@ RED.nodes = (function() {
                             subflowSet.push(n);
                         }
                     });
+                    RED.nodes.eachConfig(function(n) {
+                        if (n.z == subflowId) {
+                            subflowSet.push(n);
+                        }
+                    });
                     var exportableSubflow = createExportableNodeSet(subflowSet, exportedIds, exportedSubflows, exportedConfigNodes);
                     nns = exportableSubflow.concat(nns);
                 }
             }
             if (node.type !== "subflow") {
-                var convertedNode = RED.nodes.convertNode(node);
-                for (var d in node._def.defaults) {
-                    if (node._def.defaults[d].type) {
-                        var nodeList = node[d];
-                        if (!Array.isArray(nodeList)) {
-                            nodeList = [nodeList];
-                        }
-                        nodeList = nodeList.filter(function(id) {
-                            if (id in configNodes) {
-                                var confNode = configNodes[id];
-                                if (confNode._def.exportable !== false) {
-                                    if (!(id in exportedConfigNodes)) {
-                                        exportedConfigNodes[id] = true;
-                                        set.push(confNode);
-                                        return true;
+                if (nns.findIndex((v) => v && v.id === node.id) == -1) {
```

Comment:
As IE11 support is a requirement Hitachi have repeatedly mentioned, this code cannot use `findIndex` or `=>` style syntax.

However, this check is not needed if we add the following at line 854:
```
exportedConfigNodes[n.id] = true;
``

This object keeps tracks of the config nodes already added to the current export set - which is exactly what you need here to avoid the duplicate export.

I will make the required changes before merging.

Reply:
Thank you very much for your correction, including IE11 support.
The Config node for the UI module is now successfully exported.